content/docs/command-reference/types.md

```diff
@@ -28,7 +28,7 @@ capabilities.
 
 ## Options
 
-- `-h, --help`: Show this message and exit.
```

Comment:
does it mean that -h is not supported? why?

Reply:
probably default click/typer behavior. I will check that

Reply:
fixed